lib/internal/crypto/aes.js

```diff
@@ -252,12 +253,12 @@ async function aesImportKey(
   extractable,
   keyUsages) {
   const { name } = algorithm;
-  const checkUsages = ['wrapKey', 'unwrapKey'];
+  const usagesSet = new SafeSet(keyUsages);
+  const checkUsages = [usagesSet, 'wrapKey', 'unwrapKey'];
   if (name !== 'AES-KW')
     ArrayPrototypePush(checkUsages, 'encrypt', 'decrypt');
 
-  const usagesSet = new SafeSet(keyUsages);
-  if (hasAnyNotIn(usagesSet, ...checkUsages)) {
+  if (ReflectApply(hasAnyNotIn, null, checkUsages)) {
```

Comment:
These call sites of `hasAnyNotIn` are much harder to read now, for no real benefit.

It might be better to simply change `hasAnyNotIn` to accept an array as the second argument instead of a variable number of arguments.

Reply:
OK, I'll try that, if it doesn't have a negative perf impact I'm all for it.